bundles/org.openhab.binding.netatmo/src/main/resources/ESH-INF/thing/station.xml

```diff
@@ -58,14 +114,34 @@
 			<channel id="TempTrend" typeId="temperatureTrend" />
 			<channel id="Humidity" typeId="humidity" />
 			<channel id="Humidex" typeId="humidex" />
+			<channel id="MinHumidity" typeId="minHumidity" />
+			<channel id="MinHumidityThisWeek" typeId="minHumidityThisWeek" />
+			<channel id="MinHumidityThisMonth" typeId="minHumidityThisMonth" />
+			<channel id="MaxHumidity" typeId="maxHumidity" />
+			<channel id="MaxHumidityThisWeek" typeId="maxHumidityThisWeek" />
+			<channel id="MaxHumidityThisMonth" typeId="maxHumidityThisMonth" />
+			<channel id="DateMinHumidity" typeId="dateMinHumidity" />
+			<channel id="DateMinHumidityThisWeek" typeId="dateMinHumidityThisWeek" />
+			<channel id="DateMinHumidityThisMonth" typeId="dateMinHumidityThisMonth" />
+			<channel id="DateMaxHumidity" typeId="dateMaxHumidity" />
+			<channel id="DateMaxHumidityThisWeek" typeId="dateMaxHumidityThisWeek" />
+			<channel id="DateMaxHumidityThisMonth" typeId="dateMaxHumidityThisMonth" />
```

Comment:
Please move this block just before humidex and so just after humidity.
To be coherent with other thing types.

Reply:
done